docs/first-use/query.txt

```diff
@@ -150,7 +152,7 @@ This time, all we want to know is how many tweets we have.
 Using `HTTPie`_, you can run this query against CrateDB listening on
 ``localhost:4200``, like so:
 
-.. code-block:: sh
+.. code-block:: console
```

Comment:
where does the lexer for "console" come from?
I cannot find it here: http://pygments.org/docs/lexers/

Reply:
http://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer